### ISSUE TITLE
Allow for banner link to be in an inline/dataview field

### ISSUE THREAD

**Issue by Boonthrall** [HandaArchitect/obsidian-banner-snippet#10]
### Description/Request
Would it be possible to add the ability to have the banner link in an inline field?
#### Example
Something like this: `(cover:: ![[Fantasy grass field.webp|banner]])`
### Why
This is mostly in regards to the Minimal Theme, but I think other themes have the same functionality.
Minimal has a [Cards](https://minimal.guide/cards) feature with a definable cover. Either through a frontmatter property or inline/dataview field.
![movie-gallery](https://github.com/user-attachments/assets/9db04b73-ecd0-48c1-9f6d-0b2ef18f7343)
Allowing for the banner to be in a frontmatter/inline field would remove the necessity to set the same value in two fields. (Once for the banner and once for the cover)
### Comments
Personally I'd be fine with either, frontmatter or inline field, implementation, I'm requesting the inline field implementation because I think it would require less changes in the existing snippet.

**Comment by HandaArchitect** [comment]
It is impossible to implement this in the frontmatter as there would be no image to manipulate (this is not a plugin).

However, this should be possible in an inline field. I'll take a look into this and see what I can come up with.

**Comment by Boonthrall** [comment]
Yeah, imagined that much, wasn't sure if snippets are able to access/manipulate frontmatter as it's technically in the same file.
But inline is fine, thank you for looking into it!
